[tellstick] binding fails to start after upgrade to OH3.1.0 M4

[tellstick] binding fails to start after upgrade to OH3 M4, worked fine in M3.

## Expected Behavior
Openhab should start normally and load the Tellstick binding without errors.

## Current Behavior
After upgrade Openhab log says it cannot download file and Openhab server will not start. it gets stuck on this. mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.2.1/1.2.1_2

I removed all bindings from the addons.config and Openhab starts. I added binding one by one until the error reoccured during tellstick binding installation.

```
2021-05-03 10:05:20.372 [ERROR] [core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-binding-tellstick': Error:

	Error downloading mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.2.1/1.2.1_2

```

## Possible Solution
Workaround to remove tellstick binding from addons.config and leave the rest there. Openhab starts, but tellstick does not work.

## Steps to Reproduce (for Bugs)
1. Upgrade to OH3 M4
2. Or install tellstick binding to OH3 M4

## Context

## Your Environment
Raspberry PI4 
OH3.1.0 M4

--- a/bundles/org.openhab.binding.lametrictime/pom.xml
+++ b/bundles/org.openhab.binding.lametrictime/pom.xml
@@ -56,7 +56,7 @@
     <dependency>
       <groupId>org.apache.servicemix.specs</groupId>
       <artifactId>org.apache.servicemix.specs.activation-api-1.2.1</artifactId>
-      <version>1.2.1_2</version>
+      <version>1.2.1_3</version>
       <scope>compile</scope>
     </dependency>
     <dependency>

--- a/bundles/org.openhab.binding.tellstick/src/main/feature/feature.xml
+++ b/bundles/org.openhab.binding.tellstick/src/main/feature/feature.xml
@@ -9,7 +9,7 @@
 		<feature dependency="true">openhab.tp-jaxb</feature>
 		<bundle dependency="true">mvn:net.java.dev.jna/jna/4.5.2</bundle>
 		<bundle dependency="true">mvn:net.java.dev.jna/jna-platform/4.5.2</bundle>
-		<bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.2.1/1.2.1_2</bundle>
+		<bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.2.1/1.2.1_3</bundle>
 		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.tellstick/${project.version}</bundle>
 	</feature>
 </features>

